tests/conftest.py

```diff
@@ -85,3 +85,17 @@ def module_db():
     yield _db  # this is where the testing happens!
 
     _db.drop_all()
+
+@pytest.fixture(scope='function')
+def patched_request(mocker):
```

Comment:
Rename to make it clear that it should only apply when you need a token from the OC backend API